selfdrive/car/volkswagen/mqbcan.py

```diff
@@ -23,11 +31,23 @@ def create_lka_hud_control(packer, bus, ldw_stock_values, enabled, steering_pres
   return packer.make_can_msg("LDW_02", bus, values)
 
 
-def create_acc_buttons_control(packer, bus, gra_stock_values, counter, cancel=False, resume=False):
-  values = gra_stock_values.copy()
+def create_acc_buttons_control(packer, bus, gra_stock_values, cancel=False, resume=False):
+  values = {s: gra_stock_values[s] for s in [
+    "GRA_Hauptschalter",           # ACC button, on/off
+    "GRA_Abbrechen",               # ACC button, cancel
+    "GRA_Tip_Setzen",              # ACC button, set
+    "GRA_Tip_Hoch",                # ACC button, increase or accel
+    "GRA_Tip_Runter",              # ACC button, decrease or decel
+    "GRA_Tip_Wiederaufnahme",      # ACC button, resume
+    "GRA_Verstellung_Zeitluecke",  # ACC button, time gap adj
+    "GRA_Typ_Hauptschalter",       # ACC main button type
+    "GRA_Codierung",               # ACC button configuration/coding
+    "GRA_Tip_Stufe_2",             # unknown related to stalk type
+    "GRA_ButtonTypeInfo",          # unknown related to stalk type
+  ]}
 
   values.update({
-    "COUNTER": counter,
+    "COUNTER": (gra_stock_values["COUNTER"] + 1) % 16,
```

Comment:
if we're just passing it through, it should be fine to either use the stock counter or use our own?